shared/src/test/factories.ts

```diff
@@ -163,14 +157,29 @@ export const randomLegalStructure = (publicFiling?: {
   return LegalPublicFilings[randomIndex];
 };
 
-export const generateLicenseData = (overrides: Partial<LicenseData>): LicenseData => {
+export const generateLicenseDetails = (overrides: Partial<LicenseDetails>): LicenseDetails => {
   return {
     nameAndAddress: generateLicenseSearchNameAndAddress({}),
-    completedSearch: false,
-    items: [generateLicenseStatusItem({})],
-    status: "PENDING",
+    licenseStatus: randomElementFromArray(["PENDING", "ACTIVE", "EXPIRED"]),
```

Comment:
dust- surprises me that we don't have like universally used consts for these or like an array of states to draw on or a type or soemthing

Reply:
will update to reference a variable